appveyor.yml

```diff
@@ -2,7 +2,7 @@
 
 image: Visual Studio 2017
 
-# cache version - netcoreapp.2.0.4
+# cache version - netcoreapp.2.0.5
```

Comment:
Do we need to drop one of the caches (maybe in a separate PR)? I have seen recently that we always hit the 1GB of AppVeyor where it tries to upload it but then fails.

Reply:
touching the appveory.yml should cause the cache to rebuild.